web/impact/impact/tests/test_organization_history_view.py

```diff
@@ -333,6 +334,45 @@ def test_startup_became_finalist_no_created_at(self):
             self.assertEqual(1, len(events))
             self.assertEqual(start_date, events[0]["datetime"])
 
+    def test_startup_became_winner(self):
+        startup = StartupFactory()
+        startup_status = StartupStatusFactory(
+            program_startup_status__startup_role__name=StartupRole.SILVER_WINNER,
+            startup=startup)
+        program = startup_status.program_startup_status.program
+        role = startup_status.program_startup_status.startup_role
+        with self.login(email=self.basic_user().email):
```

Comment:
why is this in a `with`? It doesn't seem like you need this to be a context block, as there's no cleanup to do and no exceptions to catch.

Reply:
As with my other comment, I wanted this test to be consistent with the previously written tests.  The only line that I am really interested in is `self.assertEqual(role.name, events[0]["winner_level"])`.  Can strip the rest of the things out if you prefer.